zulipterminal/ui_tools/views.py

```diff
@@ -1961,9 +1961,23 @@ def __init__(
         # Get read receipt user ids list
         response = controller.model.fetch_read_receipts(message["id"])
         if response is None:
-            return
+            response = [
+                ("msg_bold", "RESPONSE CANNOT BE FETCHED AT THE MOMENT. "),
+                "\n\nPLEASE TRY AGAIN LATER",
+            ]
+            max_cols = 50
```

Comment:
Yeah it was the same for me when I tried it. Thought this was a separate issue (perhaps a more generalised solution) so didn't pursue it further and decided to focus on the uninitialised object part by handling the `None` response case. 

I'll look into it trying to see where the code is and solve that first since it might also help in other cases where response is called.

Reply:
Let's focus on the rest of the PR first, though you might open an issue for the cases we have here to avoid losing track of them.